src/registry.rs

```diff
@@ -56,7 +57,7 @@ impl PluginRegistry {
             Box::new(move |mut req: Request| {
                 let registry = self.clone();
                 let loader = loader.clone();
-                Box::pin(async move {
+                let rslt: HandlerResponse = Box::pin(async move {
```

Comment:
Sometimes `as` is dangerous, for example, `1000u128 as u8`.

All `as` where removed because `trivial_casts` was on and I am going to revert these specific changes to remove noise